Intel oneAPI 2025.2.0 segfaults on building AVX2 version of loops_arithmetic, so disable this
Patch by Simon Branford (University of Birmingham)
--- numpy/_core/meson.build.orig	2025-09-09 11:50:08.906435000 +0100
+++ numpy/_core/meson.build	2025-09-09 11:50:29.480051431 +0100
@@ -933,7 +933,7 @@
     'loops_arithmetic.dispatch.h',
     src_file.process('src/umath/loops_arithmetic.dispatch.c.src'),
     [
-      AVX512_SKX, AVX512F, AVX2, SSE41, SSE2,
+      SSE41, SSE2,
       NEON,
       VSX4, VSX2,
       VX,
